src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
@weshaggard Ok. So, then when can be start to have UWP working with this expanded API set?  Is this "uap10.1"?  If so, does that live build work?  Or is there work to do in this library to make that happen. I need to get this working in UWP for the expanded (a.k.a netstandard17.....a.k.a NET Standard 2.0).